content/en/docs/setup/install/multicluster/gateways/index.md

```diff
@@ -63,16 +63,6 @@ Cross-cluster communication occurs over the Istio gateways of the respective clu
 1. Run the following commands in **every cluster** to deploy an identical Istio control plane
     configuration in all of them.
 
-    {{< tip >}}
-    Make sure that the current user has cluster administrator (`cluster-admin`) permissions and grant them if not.
```

Comment:
Are you sure that these instructions only apply to GKE. The GKE part looks like just an example. Maybe it's safer to leave the tip in, but remove the GKE part?
```
{{< tip >}}
Make sure that the current user has cluster administrator (`cluster-admin`) permissions.
{{< /tip >}}

```

Reply:
I'm happy to leave something in, but I'm _pretty_ sure even this tip was GKE-specific. I really doubt you need `cluster-admin` permissions, I'm _guessing_ that `admin` permissions are enough. But the standard GKE setup instructions were always "give yourself cluster-admin," which probably doesn't even make sense on other platforms.

I can't work on this right now but I'll try to get back to it later today.

Reply:
@frankbu I can confirm `cluster-admin` is a GKE specific role. Something more generic may be:

`Ensure that the current user has cluster administrator privileges. Most clouds grant this by default. Please read your cloud provider's documentation if in doubt.`

Reply:
The K8s docs seem to suggest that `cluster-admin` is a [K8s builtin role](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles) - is this incorrect? I doubt it's actually needed to install Istio though (but I haven't tried).

Reply:
OK, let's just go ahead and remove the tip completely.

The prereq section of this doc already says you need authority to deploy Istio on each cluster, anything more is just noise.